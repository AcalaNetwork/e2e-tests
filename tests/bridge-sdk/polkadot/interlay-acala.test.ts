import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'interlay',
  to: 'acala',
  token: 'INTR',
})
