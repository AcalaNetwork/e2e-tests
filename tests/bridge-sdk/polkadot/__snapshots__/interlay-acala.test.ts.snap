// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'interlay' to 'acala' using bridgeSDK cross-chain 'INTR' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": "(rounded 100000)",
  "toChain": "(rounded 1.99)",
}
`;

exports[`'interlay' to 'acala' using bridgeSDK cross-chain 'INTR' > fee 1`] = `"(rounded 0.006)"`;

exports[`'interlay' to 'acala' using bridgeSDK cross-chain 'INTR' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 10,
  "fromChain": 100000,
  "toChain": 0,
}
`;
